config/shortCodes.js

```diff
@@ -13,30 +13,34 @@ async function downloadShortCode(downloadPath) {
     return `${pathPrefix}/assets/downloads/${filename}`;
 }
 
-async function imageWithClassShortcode(imagePath, cssClass, altText) {
-    const pathPrefix = process.env.BASEURL ?? '';
-    const fileType = path.extname(imagePath).replace('.', '');
-
-    const metadata = await Image(imagePath, {
-        formats: [fileType],
-        outputDir: './_site/assets/images/',
-        filenameFormat: (id, src, width, format, options) => {
-            const basename = path.basename(src, `.${format}`);
-            return `${basename}.${id}.${format}`;
-        },
-    });
-
-    const data = metadata[fileType]?.[0] ?? metadata.jpeg[0];
-
-    // _site/ is the filesystem root of the site, so we should strip that off
-    const url = data.outputPath.replace(/^_site\//i, '');
+async function imageWithClassShortcode(imagePath, cssClass, altText, uswdsInherited) {
```

Comment:
think it's worth adding a default to `uswdInherited` or best to have to pass true/false each time?

Reply:
I was being explicit about it so that if it is missing, it would throw a hard error as opposed to building with a default but having a broken asset if the default wasn't correct.

Reply:
I get it though.. the default of the classes is "no classes" and the default alt text is "no text".  My original point still stands though